### PR TITLE
Change filter select on query_filters.html to use ng-ui-select(WIP)

### PR DESCRIPTION
**Pull request by D3xx73r**
https://community.openproject.org/work_packages/19986

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://mini-thor-feature-swap-select-boxes.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://mini-thor-feature-swap-select-boxes.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

**Comment by ulferts** [comment]
Triggering linking to Work package: https://community.openproject.org/work_packages/19986


----

Will be deleted right away

**Comment by ulferts** [comment]
Hi @mini-thor, thanks for the PR. 

Keep up the good work and let us know if you need any help.

File: frontend/app/templates/work_packages/query_filters.html [hunk 1]
```diff
@@ -20,17 +20,21 @@
...
-          <option ng-repeat="option in operatorsAndLabelsByFilterType[filter.type]" value="{{ option[0] }}" ng-selected="option[0] == filter.operator" label="{{ I18n.t('js.' + option[1]) }}">{{ I18n.t('js.' + option[1]) }}</option>
-        </select>
+        <ui-select ng-model="filter.operator" theme="select2" search-enabled="false"
+          focus="$index == focusElementIndex"
+          id="operators-{{filter.name}}"
+          name="op[{{filter.name}}]"
+          style="vertical-align: top;"
+          ng-disabled="disabled">
+
+          <ui-select-match placeholder="Select a filter">
```
Comment (ulferts) [inline frontend/app/templates/work_packages/query_filters.html:30]: Probably too early to start reviewing but this placeholder needs I18n, @mini-thor.

**Comment by ulferts** [comment]
Closing because no progress is perceivable. Feel free to reopen as soon as anything new comes up.

**Pull request closed without merging**